styles/base.css

```diff
@@ -1,7 +1,10 @@
-/*
-
-Test that base.css, portfolio.css, & jQuery CDN are operational
 
+/* Test that jQuery, portfolio.css, and base.css are operational
 .blue {
   color: blue;
-}*/
+} */
+
+body {
+  background-color: rgba(51, 16, 81, 1); /* dark purple */
```

Comment:
Your dark purple background is gorgeous. But in your heading you have purple and blue nav bars, which makes them hard to read. I would suggest editing the color of your nav bar to make it easier to read against that background!